src/components/transactionsV2/walletTransactionsV2/walletTransactionsV2.test.js

```diff
@@ -53,79 +49,103 @@ describe('WalletTransactions V2 Component', () => {
+    account: accounts.genesis,
+    match: { params: { address: accounts.genesis.address } },
+    history: { push: jest.fn(), location: { search: ' ' } },
+    followedAccounts: [],
+    transactionsCount: 1000,
+    transaction: transactions[0],
+    transactions,
+    transactionsRequested: jest.fn(),
+    transactionsFilterSet: jest.fn(),
+    loadLastTransaction: jest.fn(),
+    updateAccountDelegateStats: jest.fn(),
+    addFilter: jest.fn(),
+    loading: [],
+    wallets: {},
+    peers: { options: { code: 0 } },
+    t: key => key,
+  };
 
+  beforeEach(() => {
     wrapper = mount(<Router>
         <WalletTransactionsV2 {...props} />
       </Router>, options);
   });
 
   it('renders WalletTransactionV2 Component and loads account transactions', () => {
     const renderedWalletTransactions = wrapper.find(WalletTransactionsV2);
-    expect(renderedWalletTransactions).to.be.present();
-    expect(wrapper).to.have.exactly(1).descendants('.transactions-row');
+    expect(renderedWalletTransactions).toExist();
+    expect(wrapper).toContainExactlyOneMatchingElement('.transactions-row');
   });
 
   it('click on row transaction', () => {
     const transactionPath = `${routes.transactions.pathPrefix}${routes.transactions.path}/${transactions[0].id}`;
     wrapper.find('.transactions-row').first().simulate('click');
-    expect(props.history.push).to.have.been.calledWith(transactionPath);
+    expect(props.history.push).toBeCalledWith(transactionPath);
   });
 
   describe('Onboarding Banner', () => {
-    let clock;
-
     beforeEach(() => {
       const newProps = {
         ...props,
         account: accounts['empty account'],
       };
 
-      clock = useFakeTimers({
-        now: new Date(2018, 1, 1),
-        toFake: ['setTimeout', 'clearTimeout'],
-      });
-
       wrapper = mount(<Router>
         <WalletTransactionsV2 {...newProps} />
       </Router>, options);
     });
 
-    afterEach(() => {
-      clock.restore();
-    });
-
     it('should copy address on click on banner copy and setTimeout', () => {
       const copyBtn = wrapper.find('.onboarding .copyAddress button');
-      expect(copyBtn).to.have.text('Copy');
+      expect(copyBtn).toHaveText('Copy');
       copyBtn.simulate('click');
-      expect(copyBtn).to.have.text('Copied');
-      clock.tick(3000);
-      expect(copyBtn).to.have.text('Copy');
+      expect(copyBtn).toHaveText('Copied');
+      jest.advanceTimersByTime(3000);
+      expect(copyBtn).toHaveText('Copy');
     });
 
     it('should render onboarding banner and hide when close is clicked', () => {
-      expect(wrapper).to.have.descendants('.onboarding');
+      expect(wrapper).toContainMatchingElement('.onboarding');
       wrapper.find('.onboarding .closeBtn').simulate('click');
-      expect(wrapper).to.not.have.descendants('.onboarding');
+      expect(wrapper).not.toContainMatchingElement('.onboarding');
+    });
+  });
+
+  describe.skip('Delegate Account', () => {
```

Comment:
is it useless this unit test or you have it for later?

Reply:
This one is because the delegate tab will be used on the wallet page, but not right now.